src/foam/lib/csv/CSVOutputterImpl.js

```diff
@@ -29,9 +29,9 @@ foam.CLASS({
     {
       class: 'StringArray',
       name: 'props',
-      factory: function(of) {
```

Comment:
Instead of this change, can you make this an expression? You'll need to null out the factory if you do but it feels a little more robust.

Reply:
Done.